go.mod

```diff
@@ -6,8 +6,11 @@ require (
 	github.com/FactomProject/btcutilecc v0.0.0-20130527213604-d3a63a5752ec // indirect
 	github.com/Kubuxu/gocovmerge v0.0.0-20161216165753-7ecaa51963cd
 	github.com/StackExchange/wmi v0.0.0-20181212234831-e0a55b97c705 // indirect
-	github.com/TRON-US/go-btfs-cmds v0.1.3
+	github.com/TRON-US/go-btfs-cmds v0.1.4
 	github.com/TRON-US/go-btfs-config v0.1.5
+	github.com/TRON-US/go-mfs v0.2.0
+	github.com/TRON-US/go-unixfs v0.3.0
+	github.com/TRON-US/interface-go-btfs-core v0.3.0
```

Comment:
I thought you had another update of `interface-go-btfs-core` just now? did you tag a new release?

Reply:
Yes. v0.3.1.